client/extensions/woocommerce/app/settings/shipping/save-button.js

```diff
@@ -82,38 +88,50 @@ class ShippingSettingsSaveButton extends Component {
 	};
 
 	render() {
-		const { translate, loading, site, finishedInitialSetup, isSaving } = this.props;
+		const { hasEdits, finishedInitialSetup, isSaving, loading, site, translate } = this.props;
 		const wcsEnabled = config.isEnabled( 'woocommerce/extension-wcservices' );
 
 		if ( loading || ! site ) {
 			return null;
 		}
 
-		if ( finishedInitialSetup ) {
-			return wcsEnabled ? (
-				<Button onClick={ this.save } primary busy={ isSaving } disabled={ isSaving }>
-					{ translate( 'Save' ) }
-				</Button>
-			) : null;
+		if ( finishedInitialSetup && ! wcsEnabled ) {
+			return null;
 		}
-		const label = wcsEnabled ? translate( 'Save and finish' ) : translate( "I'm Finished" );
+
+		const clickHandler = finishedInitialSetup ? this.save : this.redirect;
+		let label = translate( 'Save' );
+		if ( ! finishedInitialSetup ) {
+			label = wcsEnabled ? translate( 'Save and finish' ) : translate( "I'm Finished" );
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Finished!' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/17271-prompt-when-unsaved-changes)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).